README.md

```diff
@@ -27,7 +60,7 @@ make metadata-server
 
```

Comment:
as we are using yarn now, the json-server stuff should probably be adjusted to that (also in Makefile).

Reply:
Done.

Reply:
I'm assuming json-server should be added to the `devDependencies` in `package.json` too, right?